Makefile

```diff
@@ -1155,6 +1157,17 @@ glance_kuttl: namespace input openstack_crds openstack_storage_crds deploy_clean
 	make deploy_cleanup
 	make cleanup
 
+.PHONY: horizon_kuttl_run
+horizon_kuttl_run: ## runs kuttl tests for the horizon operator, assumes that everything needed for running the test was deployed beforehand.
+	kubectl-kuttl test --config ${HORIZON_KUTTL_CONF} ${HORIZON_KUTTL_DIR}
+
+.PHONY: horizon_kuttl
+horizon_kuttl: namespace input openstack_crds openstack_storage_crds deploy_cleanup mariadb mariadb_deploy keystone keystone_deploy infra horizon_deploy_prep horizon ## runs kuttl tests for the horizon operator. Installs openstack and openstack-storage crds, mariadb, keystone and horizon operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+	make horizon_kuttl_run
```

Comment:
I tried to run `horizon_kuttl` and it's always failing on the first try because it tries to deploy the horizon resources before the controller-manager pod is up. Since it uses the webhook to set the containerImage, we need to make sure the operator is sueccesfully installed before proceeding. I believe something like `oc wait deployment horizon-operator-controller-manager -n ${NAMESPACE} --for condition=Available --timeout=120s` should work, but @abays  will now better

Reply:
Thanks, I tried that but it doesn't seem to find the deployment. I checked the labels from the horizon-controller-manager deployment and it doesn't seem to have the `app.kubernetes.io/name=horizon-operator` but rather a `app.kubernetes.io/name=deployment` label

Reply:
Ah, fair point.  There is still some labeling standardization that needs to happen for certain operators, with this being one of them.  Ultimately they should all have something like `openstack.org/operator-name: horizon` and we would use such a label as a selector here.  I tried to change some of these in the course of the webhooks work but did not cover all them.

Reply:
Actually, Horizon already has the right format for the label mentioned above.  But other operators are missing it and then we'd need to use the standard format in the `make wait` script.